src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public enum Color {
+    White,
+    Black,
+    Grey,
+    Yellow,
```

Comment:
```suggestion
    WHITE,
    BLACK,
    GREY,
    ...
```

we should write enum elements in CAPS